cypress/integration/views/commoncontrols.view.ts

```diff
@@ -11,3 +11,5 @@ export const itemsPerPageMenuOptions = "ul[aria-labelledby='pagination-options-m
 export const expandRow = "button[aria-label=Details]";
 export const successAlertMessage = "div[aria-label='Success Alert']";
 export const duplicateNameWarning = "div[aria-label='Danger Alert']";
+export const NameHelper = "div[id=name-helper]";
```

Comment:
Have you made this variable name upper case intentionally? Otherwise please change them to lower